### ISSUE TITLE
no I2C modes with stretch image for EV3

### ISSUE THREAD

**Issue by JorgePe** [ev3dev/ev3dev#926]
stretch for EV3, snapshot 2017-07-26
apt update shows "All packages are up to date."

```
robot@ev3dev:~$ cat /sys/class/lego-port/port0/modes
auto nxt-analog ev3-analog ev3-uart other-uart
```

**Comment by dlech** [comment]
Sure enough.

**Comment by dlech** [comment]
Fix has been released in `ev3dev-1.4.0` kernel.
